js/app.js

```diff
@@ -54,6 +54,7 @@ define(['jquery', 'knockout', 'ohdsi.util', 'appConfig', 'webapi/AuthAPI', 'atla
 
 				return pageTitle;
 			});
+			self.supportURL = config.supportUrl;
```

Comment:
Where is 'config.supportUrl' defined? Is this part of another PR?

Reply:
Sorry for not clear dependencies between PRs.
**supportUrl** is a property in config.json. Please see https://github.com/OHDSI/Atlas/pull/553/files#diff-6a02156604c06becbe5fdca178766acc (line 10)